python/doc/examples/probabilistic_modeling/stochastic_processes/plot_trend_transform.py

```diff
@@ -5,40 +5,40 @@
 # %%
 # In this example we are going to estimate a trend from a field.
 #
-# We note :math:`(\underline{x}_0, \dots, \underline{x}_{N-1})` the values of
+# We note :math:`(\vect{x}_0, \dots, \vect{x}_{N-1})` the values of
```

Comment:
Much better formatting indeed.

Before:

![image](https://github.com/openturns/openturns/assets/31351465/a0c1c230-1a99-4c56-a483-90db994d4d99)

After:

![image](https://github.com/openturns/openturns/assets/31351465/6bfa8f56-bf78-4f7e-bb8a-c861c373ba13)

Thank you!